modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -801,6 +803,60 @@
         end
       end
 
+      describe "handling the 'tracked_items'" do
+        context 'it has tracked items' do
+          it "returns a claim with 'tracked_items'" do
+            with_okta_user(scopes) do |auth_header|
+              VCR.use_cassette('bgs/tracked_items/find_tracked_items') do
+                VCR.use_cassette('evss/documents/get_claim_documents') do
+                  # Is there a better way of doing this line to not have to disable rubocop?
```

Comment:
Will it let you do this?

```
VCR.use_cassette(
  'bgs/ebenefits_benefit_claims_status/find_benefit_claim_details_by_benefit_claim_id'
) do
  # blah blah blah
end
```

If not, could store the path for the cassette in a var and just reference the var in the arg to `.use_cassette`.